app/assets/javascripts/controllers/network_router/network_router_form_controller.js

```diff
@@ -41,6 +46,30 @@ ManageIQ.angular.app.controller('networkRouterFormController', ['$http', '$scope
     miqService.miqAjaxButton(url, $scope.networkRouterModel, { complete: false });
   };
 
+  $scope.addInterfaceClicked = function() {
+    miqService.sparkleOn();
+    var url = '/network_router/add_interface/' + networkRouterFormId + '?button=addInterface';
```

Comment:
Mostly I was just following convention elsewhere for similar add/remove association functionality. I can change it to just 'add'. I need to verify that this won't cause any ambiguity with the normal create 'add' form, but from a quick glance I think it will be fine. I'll update and re-test to make sure it's working with the change, though.